docs/application/tizen-studio/release-notes/4-1-release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Tizen Studio 4.1 Release Notes
+
+- Release Date: Nov 23, 2020
+
+## IDE and Tools
+
+### New Features
+
+- IDE
+  - Tizen 6.0 platform support has been added.
+  - Battery Historian tool has been added to help analyze the battery usage.
+  - Tizen Core new CLI support for Tizen 6.0 Native applications has been added.
+  - Toolchain has been updated to GCC 9.2 and LLVM 10.
+  - Support for macOS Catalina, Ubuntu 18.04, and Ubuntu 20.04 has been added.
```

Comment:
The supported OS information is updated.

For now, [the related page](https://github.com/Samsung/tizen-docs/blob/master/docs/application/tizen-studio/setup/prerequisites.md#os-and-system-requirements) was updated at previous Tizen Studio 4.0 release.
Thus the page doesn't need update at this release.

From next time, if the supported OS information is changed, please cross check the prerequisites page for Tizen Studio also to match the OS support information.

Reply:
OS Support is still the same, there is no change.

Reply:
@mijin85cho can I resolve this conversation?

Reply:
Could you change like that?
Please remove them
  - Tizen 6.0 platform support has been added.
  - Battery Historian tool has been added to help analyze the battery usage.
  - Tizen Core new CLI support for Tizen 6.0 Native applications has been added.
  - Toolchain has been updated to GCC 9.2 and LLVM 10.
  - Support for macOS Catalina, Ubuntu 18.04, and Ubuntu 20.04 has been added.

Please add them
- Update Tizen Mobile/Wearable 5.5 emulator image
- Update IoT Setup Manager